### PR TITLE
Change devs.yaml for newer links and a bio in Fery (so me)

### PR DESCRIPTION
**Pull request by NormalFery**
Pretty much a single commit I am VERY scared about because i've never seen anything that isn't .html , or .css files 
i am shaking right now

This PR literally does nothing more but add a bio for me and change the website to point to the new domain

### CODE REVIEW

**Comment by blryface** [comment]
sob

File: src/lib/vars/devs.yaml [hunk 1]
```diff
@@ -56,15 +56,18 @@ Blurryface:
...
+    Homosexual audacity, in a spanish vessel "How hard can Minecraft mod developing really be?" 
+    (Spoiler alert: very.)
   roles:
     - texture artist
   projects:
     - pridepack
     - pridegui
   socials:
-    website: https://normalfery.github.io
+    website: http://normalfery.es
```
Comment (blryface) [inline src/lib/vars/devs.yaml:68]: Maybe make your website support ssl first?

**Review by blryface — CHANGES_REQUESTED** [review]
otherwise lgtm. If it already does, just change it to https

**Comment by Ampflower** [comment]
Merging once fixing the link to use HTTPS.

**Pull request merged**